boards/components/src/storage_permissions/tbf_header.rs

```diff
@@ -12,10 +12,11 @@ use kernel::process::ProcessStandardDebug;
 
 #[macro_export]
 macro_rules! storage_permissions_tbf_header_component_static {
-    ($C:ty $(,)?) => {{
+    ($C:ty, $D:ty $(,)?) => {{
         kernel::static_buf!(
             capsules_system::storage_permissions::tbf_header::TbfHeaderStoragePermissions<
                 $C,
+                $D,
```

Comment:
This is just fixing a missing parameter that wasn't working before this PR, yes?

Reply:
Yeah, the current code doesn't compile, this just fixes it so it does compile when we go to use it